airflow/models/dagbag.py

```diff
@@ -352,7 +376,8 @@ def collect_dags(
             dag_folder=None,
             only_if_updated=True,
             include_examples=configuration.conf.getboolean('core', 'LOAD_EXAMPLES'),
-            safe_mode=configuration.conf.getboolean('core', 'DAG_DISCOVERY_SAFE_MODE')):
+            safe_mode=configuration.conf.getboolean('core', 'DAG_DISCOVERY_SAFE_MODE'),
+            dagcached_enabled=False):
```

Comment:
We should remove this. It isn't used in this function.

L393 uses: `self.dagcached_enabled`

Reply:
Yes!